lib/api/src/js/module.rs

```diff
@@ -207,16 +180,34 @@ impl Module {
     /// This is safe since the JS vm should be safe already.
     /// We maintain the `unsafe` to preserve the same API as Wasmer
     pub unsafe fn from_binary_unchecked(
-        _store: &impl AsStoreRef,
-        binary: &[u8],
+        store: &impl AsStoreRef,
+        binary: impl IntoBytes,
```

Comment:
We chatted about this before with @ptitSeb and @john-sharratt . Binary should be `&[u8]`

Reply:
I restored it to the previous state.

This was botched up during a rebase.